packages/jest-react-native/src/index.js

```diff
@@ -9,7 +9,27 @@
 /* global jest */
 
 jest
-  .mock('Image')
+  .mock('ImageViewManager', () => ({
+    prefetchImage: jest.fn(),
+    getSize: jest.fn((uri, success, failure) =>
+      process.nextTick(() => success(1, 1)),
+    ),
+  }), {virtual: true})
+  .mock('Image', () => {
+    const realImage = require.requireActual('Image');
+    const React = require('React');
+    class Image extends React.Component {
+      render() {
+        return React.createElement('Image', this.props, this.props.children);
+      }
+    }
+
+    Image.resizeMode = realImage.resizeMode;
+    Image.propTypes = realImage.propTypes;
+    Image.prefetch = realImage.prefetch;
+    Image.getSize = realImage.getSize;
```

Comment:
can you alphabetize this list? We usually try to do that :)

I also recommend doing:

```js
Object.assign(Image, {
  getSize: realImage.getSize,
  ...
});
```

to make this a bit easier on the eyes.